.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# How to Contribute
+
+## UCSD Library Development Team
+Please use the following process for creating a new [Pull Request](https://help.github.com/articles/using-pull-requests "Pull Request").
+
+1. Create a feature branch for new work.
+2. Push commits to the feature branch, with appropriate test coverage (and tests passing).
+3. Prior to creating a Pull Request, make sure your branch is up to date with its parent branch (usually `develop`)
+```
+git checkout develop
+git pull --rebase
+git checkout <your-feature-branch>
+git rebase develop
+```
+4. Run your tests again.
+5. Run rubocop `rubocop` (unless you already have it integrated into your IDE/Editor) and fix reported issues. This can be done automatically in some cases with `rubocop -a` 
```

Comment:
Robocop? For this?

Reply:
copy/paste.. you know we should probably come up with a setup of these templates for devops-y stuff